paddle/fluid/framework/operator.h

```diff
@@ -184,12 +184,122 @@ class OperatorBase {
                        const platform::Place& place) const = 0;
 };
 
+template <typename TAlgorithm>
+class AlgorithmsCache {
```

Comment:
It is not appropriate that placing `AlgorithmsCache ` in `operator.h` because it is only used by conv_op.

Reply:
moved to a header file.